src/components/ComponentPreview.tsx

```diff
@@ -1,14 +1,12 @@
+import JSONEditor from '@open-formulieren/monaco-json-editor/lib/jsonEditor';
```

Comment:
Couldn't make the `import JSONEditor from '@open-formulieren/monaco-json-editor';` import statement to work :/

Reply:
that's because you are missing the exports mapping in `package.json`: https://github.com/open-formulieren/formio-builder/blob/main/package.json#L8